README.md

```diff
@@ -1 +1,109 @@
-# angular-concepts-readme
+# Angular Core Concepts
+
+## Overview
+
+In this lesson, we are going to look at the core concepts that we will use in every Angular application. Some will be familiar to you, however some may seem a bit strange. Don't worry if they take time to sink in!
+
+## Objectives
+
+- Define MVC (Model-View-Controller), MVVM (Model-View-ViewModel)
+- Describe one-way and two-way data-binding
+- Describe Angular's two-way binding
+- Describe Dependency Injection
+- Describe a Single Page Application and common features
+- Demonstrate server-side rendering concepts
+
+## Single Page Applications
+
+Angular allows us to create single page applications (SPAs). The means that the user goes to one page and very rarely will have to refresh/leave that page to go to other parts of the website.
+
+Single page applications handle all the routing, calling different parts of your codebase dependant on what URL the user is currently at.
+
+Tranditionally, server-side rendering would be like the following -
+
+![Server-side rendering](http://www.michaelgallego.fr/images/posts/2012-11-26-client-side-1.png)
+
+The user requests the page from the server, and the server will return all the HTML to display on that page. The data would not be updated until the user refreshes. This puts overhead on the server, as it will have to render the page on every page load.
+
+With Angular, the flow will go like this instead -
+
+![Client-side rendering](http://branchandbound.net/pics/client-side-rendering-only.png)
+
+This looks a lot more complicated, but it isn't! Instead of returning rendered HTML for each user, the server would return the same page for every user. The client will then make a request to the server, requesting what's necessary for that page to render (such as a list of products). The client will then render this page for the user.
+
+This has massive advantages - if 1000 users are on the website, instead of one server rendering 1000 pages, 1000 clients are rendering one page, reducing strain on the server. We can also re-use backend endpoints - our mobile website and our desktop website can both request the same data from the server but display it differently.
```

Comment:
This makes much more sense now! Can you break it down even further? Make the example more specific - name an app where the server would send back a completely rendered template vs one where the server sends a blank template w/ data and the client renders it. Using a specific example that the reader is familiar with will make this super clear.